src/System.Diagnostics.Contracts/tests/BasicContractsTests.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.Contracts;
+using Xunit;
+
+namespace System.Diagnostics.Contracts.Tests
+{
```

Comment:
How much coverage does this actually hit in the APIs? It doesn't seem like it hits very much.